requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 pytest>=2.9.2
 tox>=2.3.1
+semantic_version
```

Comment:
Since this is a dev requirement will you pin this to either an `==` or a `>=` version.